src/Listener/EventLoggingListenerAggregate.php

```diff
@@ -22,10 +21,10 @@
  * @author Mark Garrett <mark@moderndeveloperllc.com>
  * @since 0.0.3
  */
-class EventLoggingListenerAggregate implements SharedListenerAggregateInterface
```

Comment:
@svycka Essentially, that interface goes away with v3, so to make code forwards compatible, you cannot directly implement the interface. Since this particular class implements a method for attaching listeners to the shared manager, and that method is what is called inside the `Module` class, it continues to work perfectly.